Code/Tools/FBuild/FBuildCore/Helpers/VSProjectGenerator.h

```diff
@@ -113,7 +112,6 @@ class VSProjectGenerator
                                         AString & outRelativeFileName );
 
     // project details
-    AString m_ProjectName;
```

Comment:
This wasn't used for anything other than GUID generation, if the GUID wasn't set.